allspice/utils/list_components.py

```diff
@@ -425,7 +457,14 @@ def _resolve_child_relative_path(child_path: str, parent_path: str) -> str:
     child = pathlib.PureWindowsPath(child_path)
     parent = pathlib.PureWindowsPath(parent_path)
 
-    return str(parent.parent / child)
+    relative_path = str(parent.parent / child)
+
+    # Some sheet references don't have the attribute of the doc ending
+    # with .SchDoc extension. When that is the case, add the extension.
+    if not relative_path.lower().endswith(".schdoc"):
+        relative_path += ".SchDoc"
```

Comment:
This should belong in the `_extract_repetitions` function - it being here can be confusing.